src/reference/informative/informC04_component_reference.rst

```diff
@@ -8,4 +8,189 @@
 
   .. container:: infospec
 
-    **TODO**
+    .. container:: heading3
+
+      Understanding component references
+
+    As with references to units and variables, the term "component reference" refers to places where you need to specify a component by name.
+    This is done through the :code:`component_ref` attribute found in :code:`encapsulation` and :code:`import component` elements, as well as the :code:`component_1` and :code:`component_2` attributes of :code:`connection` elements.
```

Comment:
I would probably replace
```
:code:`import component`
```
with
```
:code:`component` (within an :code:`import` element)
```

Also, I would mention
```
:code:`encapsulation`
```
after (simply because I tend to list imports before encapsulations in a CellML file), i.e. have the following in the end:
```
This is done through the :code:`component_ref` attribute found in :code:`component` (within an :code:`import` element) and :code:`encapsulation` elements, as well as the :code:`component_1` and :code:`component_2` attributes of :code:`connection` elements.
```

Reply:
I'll switch the order, but leave the `import component` as this is the name of an entire section, so want to keep it consistent with titles elsewhere. Done in bd6d599

Reply:
I find it confusing. When I first read it, I was "oh, is there something called `import component`?" and then searched for it and sure enough I couldn't find any occurrence of `import component`.

Reply:
It's a whole section?? https://cellml-specification-dev.readthedocs.io/en/i138_informative_3_4/reference/formal_and_informative/specB04.html

Reply:
No, no, I was talking about me understanding `import component` as being a CellML element of sorts, which it clearly isn't.

Reply:
Yeah, I found that idea quite confusing when I first read the spec.  Now I think we should retain the language of `import component` (and `import units`) but reinforce throughout the informative spec that this is a shorthand for component-that-has-been-imported.

Reply:
I would be happy if there was to be such a reinforcement. Actually, why not `import➤component` or something similar to distinguish the two elements?

Reply:
I think it's better to leave it as the element has been defined in section 2: two separate words, keep it simple.  Especially as the contents of the `import` is not a `component` really, it's a reference to one, so they're not strictly the same thing in any case.  If anything we should change it (everywhere) to be `imported component`, so that the language fits better, but I suspect there would be objections to that too!

Reply:
- In CellML/XML, the contents of an `import` element is a `component` element.
- `imported component` is not right either. It is rendered as some code, so I would read `imported` as being the name of a CellML element, which it isn't.

Anyway, I have voiced my view and will leave it to the others to chip in if they want.